src/Pact/Repl.hs

```diff
@@ -130,7 +130,7 @@ initPureEvalEnv verifyUri = do
 initEvalEnv :: LibState -> IO (EvalEnv LibState)
 initEvalEnv ls = do
   mv <- newMVar ls
-  gasRef <- newIORef 0
+  gasRef <- newIORef mempty
```

Comment:
```suggestion
  gasRef <- newIORef 0
```

Reply:
If this is because it's newtype-wrapped, then ignore this and all related comments.